students/boksil/users/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class User(models.Model):
+    email    = models.EmailField(max_length=45, unique=True)
+    mobile   = models.CharField(max_length=20, null=True, blank=False) # null=True
+    nickname = models.CharField(max_length=45, null=True, blank=False) # null=True
+    password = models.CharField(max_length=90) 
+
+    class Meta():
```

Comment:
상속받는 클래스가 없어서 소괄호 불필요합니다.
```suggestion
    class Meta:
```